src/service/follower.proto

```diff
@@ -25,9 +25,6 @@ message follower_gateway_req_v1 {
   bytes address = 1;
 }
 
-/// Request a stream of all active gateways from the on-chain metadata
-message follower_gateway_stream_req_v1 {}
```

Comment:
Add Bach size as a request parameter? constrain to max. < 5000 or something?

Reply:
good call; added [17fc5ff](https://github.com/helium/proto/pull/190/commits/17fc5ff2fd774d07c934d682ce4eb68124a6c699)